### PR TITLE
core: frontend: components: AutopilotManagerUpdater: Add restart function

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: core/frontend/src/components/parameter-editor/ParameterEditorDialog.vue [hunk 1]
```diff
@@ -125,15 +125,10 @@
 <script lang="ts">
 import Vue, { PropType } from 'vue'
 
+import * as AutopitlotManager from '@/components/autopilot/AutopilotManagerUpdater'
```
Comment (Williangalvani) [inline core/frontend/src/components/parameter-editor/ParameterEditorDialog.vue:128]: AutopitlotManager

**Review by Williangalvani — CHANGES_REQUESTED** [review]
nice, I was just thinking about where to place it. this looks good

found a typo though

**Comment by patrickelectric** [comment]
ping @Williangalvani @joaoantoniocardoso @rafaellehmkuhl

**Review by joaoantoniocardoso — APPROVED** [review]
Good one, tested :)

**Pull request merged**